lib/generators/shopify_app/install/templates/shopify_app.rb.tt

```diff
@@ -7,6 +7,9 @@ ShopifyApp.configure do |config|
   config.after_authenticate_job = false
   config.api_version = "<%= @api_version %>"
   config.shop_session_repository = 'Shop'
+
+  config.shop_access_scopes_strategy = ShopifyApp::AccessScopes::ShopStrategy
```

Comment:
Should we introduce the `config.shop_access_scopes` configuration in this PR?

I don't think we've documented the advanced layer anywhere yet, but this PR seems like a good place to introduce these configurations.

Reply:
We need to determine whether we want to make `shop_access_scopes_strategy` configurable right now. @tanema any thoughts on whether the strategies around access scopes should be configurable by partners? We can always make it non-configurable to begin with and then make the move to allow partners to extend it if it's desired.